src/button/component.jsx

```diff
@@ -76,6 +76,24 @@ function isCreditDualEligible(props) : boolean {
     return true;
 }
 
+function determineAPMBlacklist (allowed, disallowed) : Object {
+    Object.getOwnPropertyNames(ALTERNATE_PAYMENT_METHOD).forEach(apm => {
```

Comment:
FYI, there's for.. in: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in

Reply:
yes, but it takes in all enumerable properties, though it'll be just better to use ^ instead.

Reply:
FYI best to use: `for (let key in Object.keys(obj)) {`